src/cartridges/int_adyen_SFRA/cartridge/controllers/middlewares/adyen/notify.js

```diff
@@ -8,10 +8,10 @@ const AdyenConfigs = require('*/cartridge/scripts/util/adyenConfigs');
  */
 
 function handleHmacVerification(hmacKey, req) {
-  if (!hmacKey) {
-    return false;
+  if (hmacKey) {
+    return checkAuth.validateHmacSignature(req);
   }
-  return checkAuth.validateHmacSignature(req);
+  return true;
```

Comment:
How are we handling cases without HMAC validation? 
Should we add a log to make a merchant aware that they are not using HMAC and that we are recommending using that?